src/main/scala/system/RocketTestSuite.scala

```diff
@@ -119,14 +119,14 @@ object DefaultTestSuites {
   val rv32siNames = LinkedHashSet("csr", "ma_fetch", "scall", "sbreak", "wfi", "dirty")
   val rv32si = new AssemblyTestSuite("rv32si", rv32siNames)(_)
 
-  val rv32miNames = LinkedHashSet("csr", "mcsr", "illegal", "ma_addr", "ma_fetch", "sbreak", "scall")
+  val rv32miNames = LinkedHashSet("csr", "mcsr", "illegal", "ma_addr", "ma_fetch", "sbreak", "scall", "breakpoint", "lh-misaligned", "lw-misaligned", "shamt", "sh-misaligned", "sw-misaligned", "zicntr")
```

Comment:
I did not see a failure for rv32mi-p-zicntr

Reply:
> I did not see a failure for rv32mi-p-zicntr

Have you checked this by running it manually? The CI did not run the zicntr test because the CI only runs the regression tests; the asm-tests are not covered.

For me,

```
./out/emulator/freechips.rocketchip.system.TestHarness/freechips.rocketchip.system.TinyConfig/verilator/elf.dest/emulator $RISCV/riscv64-unknown-elf/share/riscv-tests/isa/rv32mi-p-zicntr
```
would results in
```
*** FAILED *** (tohost = 2)
*** FAILED *** via dtm (code = 2, seed 1671086616) after 52100 cycles
```

Reply:
Sorry, I was using RV32+BigRocket, not TinyRocket. 

I propose supporting the unprivileged cycle[h]/instret[h] in a M-mode only Rocket. That allows this test to pass.

Reply:
> I propose supporting the unprivileged cycle[h]/instret[h] in a M-mode only Rocket. That allows this test to pass.

Agreed. I've tested it with zicntr riscv-tests and it passed. That used to cause troubles for users, e.g. https://forums.sifive.com/t/read-csrs-rdcycle-on-rv32im/624

Since the ratification of Zicntr and Zihpm, we may also enable hpmcounter without usingUser in another PR.

Also, do we have a list of extensions that rocket supports? We may include them in isaDTS. It seems that zicsr/zicntr can be unconditionally shown there. Also zfh can be shown there with WithFP16.

Reply:
Yes, that is a good point, can you open a PR fixing isaDTS separately?

Reply:
> Yes, that is a good point, can you open a PR fixing isaDTS separately?

Will do after this gets merged.

Reply:
Thanks

Reply:
> Will do after this gets merged.

Zicntr is not fully supported now. We need to resolve https://github.com/chipsalliance/rocket-chip/issues/3207 before adding zicntr in DTS.